client/my-sites/marketing/connections/service-description.jsx

```diff
@@ -33,6 +37,38 @@ class SharingServiceDescription extends Component {
 					}
 				);
 			},
+			instagram_business: function () {
+				if ( this.props.numberOfConnections > 0 ) {
+					return this.props.translate(
+						'Sharing photos and video reels to your Instagram account.',
+						'Sharing photos and video reels to your Instagram accounts.',
+						{
+							count: this.props.numberOfConnections,
+							comment:
+								'Description for Instagram Publicize when one or more accounts are connected',
+						}
+					);
+				}
+
+				return (
+					<>
+						{ this.props.translate(
+							'Share photos and video reels from your site to your Instagram Business account.',
+							{
+								comment: 'Description for Instagram Publicize when no accounts are connected',
+							}
+						) }
+						<InfoPopover className="instagram-business__info">
+							{ this.props.translate(
+								'Instagram requires a business account connected to Facebook in order to work with third party services.'
+							) }
+							<ExternalLink className="instagram-business__help-link" href="https://wordpress.org">
```

Comment:
Shall we change this to a Jetpack Redirect, so we don't have to create another PR to update it?

Reply:
I have added a Jetpack redirect.